POKEMON_OF_THE_DAY/README.md

```diff
@@ -0,0 +1,21 @@
+# Pokémon of the Day
+
+
+## Intro
+
+Pokémon Of The Day is a simple integration built to be run on Pandium.  This is the result of Pandium's Pokémon of the Day tutorial.
```

Comment:
a link to the tutorial would be nice
I think `This is the result of Pandium's Pokémon of the Day tutorial.` sounds a little weird. maybe `This accompanies Pandium's Pokemon Of The Day tutorial` or `This is the result of following Pandium's Pokemon Of The Day tutorial.`

Reply:
eventually yes, but the tutorial will link to this Repo in more places.  I need to get this merged so I can link this in my tutorial pull request.
once the tutorial pull request is merged I can link to it in the repo.